pages/devs/get-started/cli.mdx

```diff
@@ -18,7 +18,7 @@ To install Go, follow one of the recommended methods below or consult the [offic
 ### Installation
 
 ```Text bash
-curl -sSL https://raw.githubusercontent.com/allora-network/allora-chain/main/install.sh | bash -s -- v0.2.11
+curl -sSL https://raw.githubusercontent.com/allora-network/allora-chain/main/install.sh | bash -s -- v0.2.14
```

Comment:
Is this the correct version? Just making sure this is correct since our testnet-1 is on v0.3.0 , right?

Reply:
changed to 0.3.0